service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/ClonePolicyAttributesStep.java

```diff
@@ -5,18 +5,26 @@
 import bio.terra.stairway.FlightMap;
```

Comment:
Add test for this case? Eg `cloneWorkspace_sourceDoesNotHavePao`. Could go in WorkspaceApiIControllerTest or WorkspaceApiIControllerConnectedTest, depending on if you want real or mock TPS.

Reply:
Going to leave this as is for now. I attempted both the ControllerTest and ConnectedControllerTest and they have separate issues for this case. Last time, I had to add an integration test. I started on that and it turned into a larger item, so I'll prepare a separate review for it.